autoload/slime/targets/wezterm.vim

```diff
@@ -3,7 +3,7 @@ function! slime#targets#wezterm#config() abort
   if !exists("b:slime_config")
     let b:slime_config = {"pane_id": 1}
   elseif exists("b:slime_config.pane_direction")
-    let pane_id = slime#common#system("wezterm cli get-pane-direction %s", [b:slime_config["pane_direction"]])
+    let pane_id = trim(slime#common#system("wezterm cli get-pane-direction %s", [b:slime_config["pane_direction"]]))
```

Comment:
looking good ^

can we put this on a separate line? (I think it'll be clearer)

```suggestion
    let pane_id = slime#common#system("wezterm cli get-pane-direction %s", [b:slime_config["pane_direction"]])
    let pane_id = trim(pane_id)
```

Reply:
Done!